src/app/cube/shared/featured/featured.component.html

```diff
@@ -4,7 +4,13 @@
   </a> -->
 </div>
 
-<div class="card-container">
+<div *ngIf="learningObjects.length > 0; else noObjects" class="card-container">
     <learning-object-component *ngFor="let l of learningObjects" [learningObject]="l" learningObjectCard [loading]="loading">
     </learning-object-component>
 </div>
+
+<ng-template #noObjects>
```

Comment:
A good practice here (personal opinion incoming) is to add the word `template` to the end of ng-templates you create in the markup. This is because, under the hood,  Angular creates a javascript variable for it, in this case called `noObjects` and they have a tendency to conflict. Just a suggestion, learned this the hard way myself. I also personally think it reads a little easier.

Reply:
Appended "Template" to the end of the template.